src/index.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './globalScss/index.scss';
-import App from './pages/App/App';
+// import App from './pages/App/App';
+import UnsupportedFileFormat from './components/UnsupportedFileFormat/UnsupportedFileFormat.component';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+// ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<UnsupportedFileFormat />, document.getElementById('root'));
```

Comment:
I don't think you should add an Unsupported file format in index.js. It should be put as a component in-app.js